### PR TITLE
Monthly translation update for OAG (Aug '23)

### PR DESCRIPTION
**Pull request by eng-prod-CI-bot-okta**
OAG translation for the latest commit: https://github.com/okta/okta-help/commits/docs_translations_OAG

### CODE REVIEW

File: oag/ja-jp/content/topics/access-gateway/deploy-msa-cli.htm [hunk 1]
```diff
@@ -302,7 +302,7 @@ <h2><a name="Prerequisites"></a>開始する前に</h2>
                                                     <div>
                                                         <h2><a name="InstallCLI"></a>コマンドラインインターフェイスのインストール</h2>
                                                         <ol>
-                                                            <li value="1">Microsoft Azure CLI概要<a href="https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest">ページ</a>に移動します。 </li>
+                                                            <li value="1">Microsoft Azure CLI概要<a href="https://docs.microsoft.com/ja-jp/cli/azure/?view=azure-cli-latest">ページ</a>に移動します。 </li>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/access-gateway/deploy-msa-cli.htm:305]: Sweet! Here's our l10n fix for external links

File: oag/ja-jp/content/topics/releasenotes/oag/oag-release-notes.htm [hunk 2]
```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/releasenotes/oag/oag-release-notes.htm:1]: Drat, this is the older version, from a previous template. Will need to update this here and on the LL branch.

File: oag/ja-jp/content/topics/releasenotes/oag/oag-version-history.htm [hunk 3]
```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/releasenotes/oag/oag-version-history.htm:1]: Drat, this is the older version, from a previous template. Will need to update this here and on the LL branch.

Comment (emikhasyak-okta) [reply oag/ja-jp/content/topics/releasenotes/oag/oag-version-history.htm:1]: If PR does not include additional commits, It's safer to update LL branch and re-run the receiving script

**Review by emikhasyak-okta — APPROVED** [review]
We still have not received the translation to the non-gh LL branch.
I'm approving PR. It's up to you if you want to merge it or wait for the translations 🚢

**Pull request merged**